.rollup.config.umd.polyfilled.js

```diff
@@ -0,0 +1,20 @@
+import commonjs from 'rollup-plugin-commonjs';
+import nodeResolve from 'rollup-plugin-node-resolve';
+
+export default {
+  entry: 'src/shopify-polyfilled.js',
+
+  format: 'cjs',
```

Comment:
No initially we bundle to commonjs then do the umd build from cjs. I'd like to try and see what would happen if we switch cjs to es6.